source/transactions/transactions.rst

```diff
@@ -438,9 +438,9 @@ have been performed on this transaction, drivers MUST NOT run the
 commitTransaction command.
 
 commitTransaction is a retryable write command. Drivers MUST retry once
-after commitTransaction fails with a retryable error according to the
-Retryable Writes Specification, regardless of whether retryWrites is set
-on the MongoClient or not.
+after commitTransaction fails with a retryable error, including a
+handshake network error, according to the Retryable Writes Specification,
+regardless of whether retryWrites is set on the MongoClient or not.
```

Comment:
This should be accompanied with a last modified and changelog update. I suppose the version can be a patch bump, since it's a subtle change (vs. the pseudo-code and implementation being updated in the retryable read/write specs).

Reply:
done! and patch sgtm